src/controllers/userController.js

```diff
@@ -0,0 +1,23 @@
+const models = require('../models');
+export const user = {
```

Comment:
Instead of exporting just an object, I think it may be better to create a class that would hold all the user controller methods and export that class by default.